tests/clv/models/test_pareto_nbd.py

```diff
@@ -290,6 +383,8 @@ def test_expected_purchase_probability(self, test_n, test_t):
             rtol=0.001,
         )
 
+        self.model.expected_purchase_probability(test_n, test_t, self.data)
```

Comment:
Did you mean `self.covar_model`? Also need to test something about the output

Reply:
`xarray` issue needs to be resolved first (see my latest comment).